readme.txt

```diff
@@ -32,7 +32,14 @@ Resources packed within the theme.
   Logo: Own Creation
   Copyright © 2016, ThemeGrill, GPL v3 or later, http://www.gnu.org/licenses/gpl-3.0.txt
   
-  Screenshot, Creative Commons Zero (CC0), https://pxhere.com/ link.
+  Screenshot, Creative Commons Zero (CC0).
+  https://pxhere.com/en/photo/914603
+  https://pxhere.com/en/photo/869156
```

Comment:
It is not shown in the screenshot so it is not necessary to mention it here. Please check for the similar cases.